tests/data/Calculation/DateTime/DAYS.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
```

Comment:
Tests cases miss a few cases for different input type such as DateTime, int and float